src/Host.cpp

```diff
@@ -345,6 +345,7 @@ Host::Host(int port, const QString& hostname, const QString& login, const QStrin
 , mRoomSize(0.5)
 , mShowInfo(true)
 , mBubbleMode(false)
+, mMapViewOnly(true)
```

Comment:
We plan to move away from initialising the elements that have constant initialisers like this - instead we are going to *eventually* get around to putting them in the header file which either the C++14 or C++17 standard now permits.

So you do not need to have this here, instead you just have to change the line in the header file:
```diff
-     bool mMapViewOnly;
+     bool mMapViewOnly = true;
```